manifests/rhoai/training-hub-universal-cuda-imagestream.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: training-hub-universal-cuda
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/trainer"
+    opendatahub.io/notebook-image-name: "Training Hub Universal (CUDA, Python 3.12)"
```

Comment:
I would remove the version from here and include torch. 
Reason why version IMO should be removed is because we can have a single imageStream with multiple version (imageStreamTags) and these versions can have different python version over time.
Unless, we can agree to keep this versioned drop it once new version (with updated python version is out)

So basically an option between core image > versions of the core image vs potentially multiple core image with different versions.